src/dash/DashAdapter.js

```diff
@@ -1044,11 +1048,15 @@ function DashAdapter() {
         mediaInfo.representationCount = dashManifestModel.getRepresentationCount(realAdaptation);
         mediaInfo.labels = dashManifestModel.getLabelsForAdaptation(realAdaptation);
         mediaInfo.lang = dashManifestModel.getLanguageForAdaptation(realAdaptation);
-        viewpoint = dashManifestModel.getViewpointForAdaptation(realAdaptation);
-        mediaInfo.viewpoint = viewpoint ? viewpoint.value : undefined;
         mediaInfo.segmentAlignment = dashManifestModel.getSegmentAlignment(realAdaptation);
         mediaInfo.subSegmentAlignment = dashManifestModel.getSubSegmentAlignment(realAdaptation);
-        mediaInfo.accessibility = dashManifestModel.getAccessibilityForAdaptation(realAdaptation).map(function (accessibility) {
+
+        viewpoint = dashManifestModel.getViewpointForAdaptation(realAdaptation);
+        mediaInfo.viewpoint = viewpoint.length ? viewpoint[0].value : undefined;
+        mediaInfo.viewpoint_withSchemeIdUri = viewpoint;
```

Comment:
Suggest to rename to plural `viewpointsWithSchemeIdUri`. Compared to the `viewpoint` attribute we might have multiple entries here right?

Reply:
Indeed, as per the DASH spec, this descriptor has an usage of '0..N'. Will change this!

_However:_ 
Grepping through 23009-1, I don't find a "viewpoint attribute".
IMHO, the conversion (in line 1055, one line above) from the `viewpoint` array (as returned from `dashManifestModel.getViewpointForAdaptation()`) into a single value in `mediaInfo.viewpoint` is already a bug.
Still, I'm rather hesitant to change this now, because:
1. this would be a non-backward-compatible change
2. this we might deprecate this element (in a future release - tbd).

Reply:
Agreed `viewpoint[0].value` is already wrong. Changing this might impact existing applications but I think we should still consider making this change. At least if we decide to deprecate the way `viewpoint`, `role`, `accessibility`, `audioChannelConfiguration` are saved today (see below).

Reply:
2nd option is to wait with these changes for V5.0, but don't apply with V4.x - let's discuss tomorrow!

Reply:
During our monthly call we agreed to include the breaking changes in v.5.0.0